controllers/volsync/vshandler.go

```diff
@@ -946,16 +930,16 @@ func (v *VSHandler) EnsurePVCforDirectCopy(ctx context.Context,
 		return fmt.Errorf("capacity must be provided %v", rdSpec.ProtectedPVC)
 	}
 
-	exists, err := v.pvcExists(rdSpec.ProtectedPVC.Name)
+	exists, pvc, err := v.pvcExists(rdSpec.ProtectedPVC.Name)
 	if err != nil {
 		return err
 	}
 
 	if exists {
-		return nil
+		return v.removeOCMAnnotationsAndUpdate(pvc)
```

Comment:
Looks like this replace the removal we had in takeOnwership, but not clear why do we need to do this here, and if this function is called only for the secondary pvc.

Reply:
That's right. When we are moving the Primary PVC to secondary, we don't want the secondary PVC to have OCM annotations.  OCM on the secondary has nothing to do with that PVC.